common/scala/src/main/scala/whisk/connector/kafka/KafkaConsumerConnector.scala

```diff
@@ -90,7 +90,7 @@ class KafkaConsumerConnector(
         val props = new Properties
         props.put(ConsumerConfig.GROUP_ID_CONFIG, groupid)
         props.put(ConsumerConfig.BOOTSTRAP_SERVERS_CONFIG, kafkahost)
-        props.put(ConsumerConfig.SESSION_TIMEOUT_MS_CONFIG, 10000.toString)
+        props.put(ConsumerConfig.SESSION_TIMEOUT_MS_CONFIG, 30000.toString)
```

Comment:
I merged this change in https://github.com/openwhisk/openwhisk/commit/fabd87f6187066da3158c8cd17a560b81353b622.